xml/containers-support.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha-containers-support"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Supportability</title><date>2020-07-27</date>
+ </info>
+ <para>
+  This chapter explains how &slsa; container support plans work.
+ </para>
+ <para>
+  There are three guiding principles that &suse; container supportability
+  generally adheres to:
+ </para>
+ <orderedlist numeration="arabic">
+  <listitem>
+   <para>
+    The container image lifecycle follows the lifecycle of the related
+    products.
+   </para>
+   <para>
+    For example, &slsa; &productnumber; container images will adhere to the
+    &slsa; &productnumber; lifecycle.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Container release status also matches the status of the related product.
+   </para>
+   <para>
+    For example, if &slsa; &productnumber; is in Alpha, Beta, RC or GA stage,
+    the related containers also have that release status.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Containers are built only using the packages from the related products.
+   </para>
+   <para>
+    For example, &slsa; &productnumber; container images will be built using
+    the same packages as the main &slsa; &productnumber; release.
+   </para>
+  </listitem>
+ </orderedlist>
+ <sect1 xml:id="id-supported-containers-on-suse-host-environments">
+  <title>Supported Containers on SUSE Host Environments</title>
+
+  <para>
+   The following support options (Tiers) apply to
```

Comment:
"Tier" is randomly capitalized in the following section, you can just lower-case it, methinks.